hibernate-core/src/main/java/org/hibernate/bytecode/spi/BytecodeEnhancementMetadata.java

```diff
@@ -59,4 +59,6 @@ LazyAttributeLoadingInterceptor injectInterceptor(
 	LazyAttributeLoadingInterceptor extractInterceptor(Object entity) throws NotInstrumentedException;
 
 	boolean hasUnFetchedAttributes(Object entity);
```

Comment:
Because EntityPersister#getInstrumentationMetadata returns BytecodeEnhancementMetadata. If the custom EntityPersister extends AbstractEntityPersister, then it's OK.

I know it's unlikely to be a problem; just mentioning it.